sensor/kubernetes/listener/resources/store_provider.go

```diff
@@ -0,0 +1,30 @@
+package resources
+
+import (
+	"github.com/stackrox/rox/sensor/common/store"
+	"github.com/stackrox/rox/sensor/kubernetes/listener/resources/rbac"
+)
+
+// InMemoryStoreProvider holds all stores used in sensor and exposes a public interface for each that can be used outside of the listeners.
+type InMemoryStoreProvider struct {
+	serviceStore *serviceStore
+	rbacStore    rbac.Store
```

Comment:
Why not 

```suggestion
	rbacStore    *rbac.Store
```

to avoid copying when calling `RBAC()`?

Reply:
Because `rbac.Store` is an interface